tests/yml/RAMLLintBearTest.py

```diff
@@ -1,4 +1,4 @@
-from bears.yml.RAMLLintBear import RAMLLintBear
+from bears.yaml.RAMLLintBear import RAMLLintBear
```

Comment:
@meetmangukiya Thanks for reviewing.

> Why not rename the test directory as well, then ?

Sure!

> Also as gitmate said, please reduce the length of shortlog to 50 chars

It is already, the comment refers to an older commit.